Automattic/wp-calypso
# Playwright: refactor EditorToolbarComponent and EditorNavSidebarComponent out of GutenbergEditorPage.

#### Changes proposed in this Pull Request

This PR refactors two new components out of GutenbergEditorPage:
- EditorToolbarComponent
- EditorNavSidebarComponent

Handling the top toolbar interactions and the editor navigation sidebar respectively.

Key changes:
- extract methods related to toolbar and navigation sidebar out of GutenbergEditorPage.
- update existing methods in GutenbergEditorPage to point to the extracted classes.

#### Testing instructions

Ensure that all following builds are green:
  - [x] WPCOM E2E (desktop)
  - [x] WPCOM E2E (mobile)
  - [x] Pre-Release

Related to https://github.com/Automattic/wp-calypso/issues/60868.
Depends on https://github.com/Automattic/wp-calypso/pull/61431.
